src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -4464,6 +4464,312 @@ Activate by holding the key for the character you want to add an accent to, then
+    <value>Bottom left corner</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_BottomCenter.Content" xml:space="preserve">
+    <value>Bottom center</value>
+  </data>
+  <data name="ZoomIt_Break_TimerPosition_BottomRightCorner.Content" xml:space="preserve">
+    <value>Bottom right corner</value>
+  </data>
+  <data name="ZoomIt_Break_ShowBackgroundBitmap.Header" xml:space="preserve">
+    <value>Show Background Bitmap</value>
+  </data>
+  <data name="ZoomIt_Break_ShowFadedDesktop.Content" xml:space="preserve">
+    <value>Use faded desktop as background</value>
+  </data>
+  <data name="ZoomIt_Break_ShowImageFile.Content" xml:space="preserve">
+    <value>Use image file as background</value>
+  </data>
+  <data name="ZoomIt_Break_BackgroundFile.Header" xml:space="preserve">
+    <value>Background Image File</value>
+  </data>
+  <data name="ZoomIt_Break_BackgroundFile_BrowseButton.Content" xml:space="preserve">
+    <value>Browse</value>
+  </data>
+  <data name="ZoomIt_Break_BackgroundFile_Picker_Dialog_Title" xml:space="preserve">
+    <value>Specify background file...</value>
+  </data>
+  <data name="FilePicker_ZoomIt_BitmapFilesFilter" xml:space="preserve">
+    <value>Bitmap Files</value>
+  </data>
+  <data name="FilePicker_ZoomIt_AllPicturesFilter" xml:space="preserve">
+    <value>All Picture Files</value>
+  </data>
+  <data name="ZoomIt_Break_BackgroundStretch.Header" xml:space="preserve">
+    <value>Scale to screen</value>
+  </data>
+  <data name="ZoomIt_RecordGroup.Header" xml:space="preserve">
+    <value>Record</value>
+  </data>
+  <data name="ZoomIt_RecordGroup.Description" xml:space="preserve">
+    <value>Record video of the unzoomed live screen or a static zoomed session by entering the recording hot key and finish the recording by entering it again.
+
+To crop the portion of the screen that will be recorded, enter the hotkey with the Shift key in the opposite mode.
+
+To record a specific window, enter the hotkey with the Alt key in the opposite mode.</value>
+  </data>
+  <data name="ZoomIt_Record_Shortcut.Header" xml:space="preserve">
+    <value>Record Toggle Hotkey</value>
+  </data>
+  <data name="ZoomIt_Record_Scaling.Header" xml:space="preserve">
+    <value>Scaling</value>
+  </data>
+  <data name="ZoomIt_Record_CaptureAudio.Header" xml:space="preserve">
+    <value>Capture audio input</value>
+  </data>
+  <data name="ZoomIt_Record_Microphone.Header" xml:space="preserve">
+    <value>Microphone</value>
+  </data>
+  <data name="ZoomIt_Record_Microphones_Default_Name" xml:space="preserve">
+    <value>Default</value>
+  </data>
+  <data name="ZoomIt_SnipGroup.Header" xml:space="preserve">
+    <value>Snip</value>
+  </data>
+  <data name="ZoomIt_SnipGroup.Description" xml:space="preserve">
+    <value>Copy a region of the screen to the clipboard or enter the hotkey with the Shift key in the opposite mode to save it to a file.</value>
```

Comment:
what if I set hotkey to Shift+<some key> ? then save to file is unreachable?

Reply:
I've tried setting it to Shift+1. Pressing just 1 will save it to a file, similar to original ZoomIt behavior.